examples/article-access.amp.html

```diff
@@ -9,7 +9,8 @@
     {
       "authorization": "http://localhost:8002/amp-authorization.json?rid=READER_ID&url=CANONICAL_URL&_=RANDOM",
       "pingback": "http://localhost:8002/amp-pingback?rid=READER_ID&url=CANONICAL_URL&views=AUTHDATA(views)",
-      "login": "http://localhost:8002/amp-login?rid=READER_ID&url=CANONICAL_URL"
+      "login": "http://localhost:8002/amp-login?rid=READER_ID&url=CANONICAL_URL",
+      "authorizationFallback": {"error": true}
```

Comment:
Sounds good. It's not `authorizationFallbackResponse`. A bit wordy, but at least it's unambiguous.